packages/pg-v5/test/lib/psql.js

```diff
@@ -184,7 +219,7 @@ describe('psql', () => {
           fakePsqlProcess.start()
           return fakePsqlProcess
         })
-        const promise = psql.exec(bastionDb, 'SELECT NOW();', 1000)
+        const promise = psql.exec(bastionDb, 'SELECT NOW();')
```

Comment:
New to me - but it appears that extra args passed to a function in js are ignored. From my review, this `1000` was being ignored and never used. Since it was interfering with the `cmdArgs` optional argument introduced in this PR, I removed these unused `1000` args in test.